src/css.js

```diff
@@ -10,17 +10,16 @@ define([
 	"./css/var/getStyles",
 	"./css/curCSS",
 	"./css/adjustCSS",
-	"./css/defaultDisplay",
 	"./css/addGetHookIf",
 	"./css/support",
-	"./data/var/dataPriv",
+	"./css/showHide",
```

Comment:
Can we pull `.show()` and `.hide()` into this file and remove the dependency inside css.js completely? That way it becomes a separate module and can be removed in a custom build if you use classes for showing and hiding.